skopt/optimizer/optimizer.py

```diff
@@ -333,13 +334,27 @@ def ask(self, n_points=None, strategy="cl_min"):
         for i in range(n_points):
             x = opt.ask()
             X.append(x)
+
+            ti = [t for (_, t) in opt.yi] if "ps" in self.acq_func else None
+
             if strategy == "cl_min":
                 y_lie = np.min(opt.yi) if opt.yi else 0.0  # CL-min lie
+                t_lie = np.min(ti) if ti else sys.float_info.max
```

Comment:
Nitpick: in python `bool([]) == False` as well as `bool(None) == False` so the recommendation is to test `if ti is not None` if that is what you mean, instead of `if ti`. In this case we might end up with a hard to spot bug if `ti = []` for some crazy reason. (same for the below)

Reply:
Not a nitpick, that's a valid issue. Thanks for the correction, always happy to learn more. I've developed most of my code in Java, so I still have to learn a lot of the python specifics.